src/actions/actionTypes.js

```diff
@@ -62,6 +62,13 @@ export const CLUSTERS_LOAD = 'CLUSTERS_LOAD';
 export const CLUSTERS_LOAD_SUCCESS = 'CLUSTERS_LOAD_SUCCESS';
```

Comment:
should these be renamed to V4 now?

Reply:
Yes, have included it in the next PR where I refactored this a little tiny bit.
While adding node pools actions I realized that we have actions without `case` in reducer, I removed them so we will be just using `LOAD_SUCCES` and `LOAD_ERROR` actions for clusters and node pools.